shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -672,6 +675,38 @@ blink::WebCacheResourceTypeStats GetResourceUsage(v8::Isolate* isolate) {
   return stats;
 }
 
+bool SpellCheckWord(v8::Isolate* isolate, 
```

Comment:
Can you move this to anonymous namespace? i.e. where the `GetRenderFrame` are defined.